yarn.lock

```diff
@@ -5,21 +5,18 @@
 "@babel/code-frame@7.0.0-beta.51":
   version "7.0.0-beta.51"
   resolved "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.0.0-beta.51.tgz#bd71d9b192af978df915829d39d4094456439a0c"
-  integrity sha1-vXHZsZKvl435FYKdOdQJRFZDmgw=
```

Comment:
Curios, why is the `integrity` getting removed? Maybe a older yarn version?

Reply:
Yeah I believe yarn 1.10+ adds the integrity hashes, I've been switching to 1.13 for ebayui-core work

Reply:
Let me double check.

Reply:
Yeah, that's way too many changes. Let me clean it up.

Reply:
Good catch there, @PrashantAshok! It should all be sorted now.